src/cli/tc.rs

```diff
@@ -104,11 +108,37 @@ pub async fn init(file_path: &str) -> Result<()> {
 pub fn display_schema(blueprint: &Blueprint) {
   Fmt::display(Fmt::heading(&"GraphQL Schema:\n".to_string()));
   let sdl = blueprint.to_schema();
-  Fmt::display(print_schema::print_schema(sdl));
+  Fmt::display(format!("{}\n", print_schema::print_schema(sdl)));
 }
 
 fn display_config(config: &Config, n_plus_one_queries: bool) {
-  Fmt::display(Fmt::success(&"No errors found".to_string()));
   let seq = vec![Fmt::n_plus_one_data(n_plus_one_queries, config)];
   Fmt::display(Fmt::table(seq));
 }
+
+fn validate_operations(blueprint: &Blueprint, operations: Vec<String>) -> Result<()> {
+  tokio::runtime::Runtime::new()?.block_on(async {
+    let schema = blueprint.to_schema();
+    let mut execution = vec![];
+
+    for op in operations.iter() {
+      if let Ok(operation) = tokio::fs::read_to_string(op).await {
```

Comment:
Instead of adding an `if else` block here to check if it is ok, can you use `unwrap_or_else` instead? it'd make it more readable

Reply:
By design, I exit if we fail to read the operation file. There is nothing to provide if we cannot read the op, I think.

Reply:
the example above is not right, `unwrap_or_else` is used to provide defaults if we have an error and will not end the function with the result. 

The current code is valid if we want to fail fast on errors. To simplify it we can use something like this:

```
let value = func().map_err(...)?;
```